internal/pkg/service/cli/dependencies/local.go

```diff
@@ -50,7 +50,7 @@ func newPublicDeps(baseDeps Base) (*local, error) {
 	}
 
 	// Create common local dependencies
-	publicDep, err := dependencies.NewPublicDeps(baseDeps.CommandCtx(), baseDeps, host, true)
+	publicDep, err := dependencies.NewPublicDeps(baseDeps.CommandCtx(), baseDeps, host, dependencies.WithComponents(true))
```

Comment:
Default value by som dal `true` a teda to tu byt nemusi (alebo false ak ti to dava vacsiu logiku).
Pri `default = false` by sa to mohlo volat `WithPreloadComponents()`.

Reply:
Ako to po sebe citam, tak to asi napisem podrobnejsie :)

Reply:
Teraz, ked nic nezadas, tak je `includeComponents=false`, teda nerobi `preload`.

1. To je myslim, celkom logicke, len nevidim dovod potom davat na dane miesta `WithComponents`.

2. A ako nad tym rozmyslam, tak mozno by bolo este lepsie `WithPreloadComponents()` (bez argumentu), alebo ``WithPreloadComponents(bool)`.

Ale to je uz na tebe, ci ti to dava zmysel alebo nie.